zkapp/utils.ts

```diff
@@ -69,6 +70,9 @@ async function deployContract(
   let transaction = await Mina.transaction(
     { sender, fee: transactionFee },
     async () => {
+      if (fundNewAccount) {
+        AccountUpdate.fundNewAccount(sender);
```

Comment:
This line is required in general, but must be omitted if we deploy to a keypair that already exists, as is the case in `tests/resolvers.test.ts#102`